tools/mock_node/src/main.rs

```diff
@@ -120,25 +120,44 @@ fn main() -> anyhow::Result<()> {
             args.target_height,
             args.in_memory_storage,
         );
+        let ping_handle = actix::spawn(async move {
+            let mut interval = tokio::time::interval(Duration::from_millis(100));
+            interval.set_missed_tick_behavior(tokio::time::MissedTickBehavior::Skip);
+            loop {
+                interval.tick().await;
+
+                let latency = {
+                    let t = Instant::now();
+                    let _ = client.send(Status { is_health_check: false, detailed: false }).await;
+                    t.elapsed()
+                };
+
+                if latency > Duration::from_millis(100) {
```

Comment:
maybe we could collect these latencies, and print the summary at the end (of how many were missed + AVG processing time)

Reply:
I'd rather not to, at least not right now:

* given that pretty much every ping takes more than 100ms for full blocks, I don't think we need more fine-grained visibility than just threshold cutoff
* I am not entirely sure that we care about latency distribution to such an extent that people usually care about those things